pkgs/development/python-modules/pip/default.nix

```diff
@@ -36,14 +43,48 @@ buildPythonPackage rec {
     installShellFiles
     setuptools
     wheel
+
+    # docs
+    sphinx
+    sphinx-copybutton
+    sphinx-inline-tabs
+    sphinxcontrib-towncrier
+    myst-parser
+  ];
+
+  sphinxBuilders = [
+    "man"
+  ];
+
+  outputs = [
+    "out"
+    "man"
   ];
 
+  postBuild = ''
```

Comment:
Perhaps I'd comment here that using `sphinxHook` is not easier then running these commands yourself, due to pip putting docs in non expected locations.